projects/packages/forms/src/contact-form/class-feedback-field.php

```diff
@@ -328,6 +328,68 @@ public static function from_serialized( $data ) {
 		);
 	}
 
+	/**
+	 * Normalize Unicode characters in a string.
+	 *
+	 * This is only used for V2 version of the feedback. Since we didn't escape special characters
+	 *
+	 * @param string $string The string to normalize.
+	 *
+	 * @return string
+	 */
+	public static function normalize_unicode( $string ) {
+		// Case 1: JSON-style escapes, e.g. "\u003cstrong\u003e" or "\ud83d\ude48"
+		if ( strpos( $string, '\u' ) !== false ) {
+			$decoded = json_decode( '"' . $string . '"' );
```

Comment:
we could extract

```			$decoded = json_decode( '"' . $string . '"' );
			if ( $decoded !== null && json_last_error() === JSON_ERROR_NONE ) {
				return $decoded;
			}```

into it's own method as we do the same in the following if?

Reply:
Good call. We can do this in a follow up PR.